mobile/keycloak/.eslintignore

```diff
@@ -0,0 +1,3 @@
+build
+dist
+example-app
```

Comment:
I think `example-app` can be removed.

Reply:
https://github.com/bcgov/wps/pull/4704/commits/192347099370c7b0f6c38e7c2001c2b183584896